app/models/form/page.rb

```diff
@@ -1,24 +1,21 @@
 class Form::Page
-  attr_accessor :id, :header, :description, :questions, :soft_validations,
-                :depends_on, :subsection, :hide_subsection_label
+  attr_accessor :id, :header, :description, :questions,
+                :depends_on, :title_text, :informative_text, :subsection, :hide_subsection_label
 
   def initialize(id, hsh, subsection)
     @id = id
     @header = hsh["header"]
     @description = hsh["description"]
     @questions = hsh["questions"].map { |q_id, q| Form::Question.new(q_id, q, self) }
     @depends_on = hsh["depends_on"]
+    @title_text = hsh["title_text"]
+    @informative_text = hsh["informative_text"]
```

Comment:
Is this different from `description`?

Reply:
the `@title_text` is for passing to the `title_text` param of the GovUK Panel component in  app/views/form/_interruption_screen_question.html.erb 

`@informative_text` was the the text under the panel `title_text` - the thinking behind it was to keep the params distinct and easily identifiable and also to have these as things only relevant to the `interruption_screen` question type